### PR TITLE
Main component

### PR DESCRIPTION
**Pull request by mrfong**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,10 +36,16 @@
...
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.17.2",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
+    "eslint-plugin-react": "^7.13.0"
+  },
   "scripts": {
-    "start": "webpack-dev-server --open --hot --mode development",
-    "build": "webpack --mode production",
-    "test": "jest"
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (misterrodger) [inline package.json]: The script/build etc needs to stay, thank you

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1 @@
+{ "presets": ["@babel/preset-env", "@babel/preset-react"]}
```
Comment (misterrodger) [inline .babelrc]: Please delete this file, it is now babel.config.js with different syntax and will fix the problem

File: package.json (hunk 1 above)
Comment (mrfong) [reply package.json]: Fixed

File: .babelrc (hunk 2 above)
Comment (mrfong) [reply .babelrc]: done

**Pull request merged**